src/api/ApiClient.ts

```diff
@@ -72,6 +78,40 @@ Client.interceptors.request.use(async (config: AxiosRequestConfig) => {
   const guestToken = await getData(LocalStorageKeys.AUTH_GUEST_TOKEN)
   const accessToken = await getData(LocalStorageKeys.ACCESS_TOKEN)
 
+  const switchGetDeviceType = () => {
```

Comment:
I don't think these two functions belong to this file. Can you please move it to a Device helper file in helpers folder?
Also please rename those names, remove the switch word from their name and leave it just like getDeviceType